packages/turbo/README.md

```diff
@@ -78,6 +78,82 @@ URL for the WKWebview to open. Changing the url should result in view replacing
 
 Session handle for the webview. If not provided, the default session will be used. It can be used to create separate webview instances for different parts of the app.
 
+### `applicationNameForUserAgent`
+
+The name of the application as used in the user agent string.
+
+### `stradaComponents`
+
+`VisitableView` supports defining [Strada components](https://strada.hotwired.dev/) that receive and reply to messages from web components that are present on the page within one session. This prop accepts an array of Strada components that will be registered in the webview.
+
+You can define a Strada component by:
+
+1. Extending `BridgeComponent` class and implementing `onReceive` method and static `name` property.
+2. Wrapping the component with `bridgeComponent` utility.
```

Comment:
I get that we have this, but I'm wondering if it should be officially exported by the library? It will need to be maintained going forward and I think the class definitions will be the preferred and recommended solution to keep closer to Strada native.

Reply:
Removing the `bridgeComponent` would also allow us to clean up this: `const Component = stradaComponent.StradaComponent ?? stradaComponent;` etc.

Thinking about it I'm certain it would provide a clearer maintenance path to only support class components.